modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -1027,20 +1031,21 @@ private <T> CompletableFuture<T> doWithOneOffRaftGroupService(
             PeersAndLearners raftClientConfiguration,
             Function<RaftGroupService, CompletableFuture<T>> action
     ) {
-        return startOneOffRaftGroupService(raftClientConfiguration)
-                .thenCompose(raftGroupService -> action.apply(raftGroupService)
-                        .whenComplete((res, ex) -> raftGroupService.shutdown())
-                );
-    }
-
-    private CompletableFuture<RaftGroupService> startOneOffRaftGroupService(PeersAndLearners newConfiguration) {
         try {
-            return raftMgr.startRaftGroupService(MetastorageGroupId.INSTANCE, newConfiguration);
+            RaftGroupService raftGroupService = startOneOffRaftGroupService(raftClientConfiguration);
+
+            return action.apply(raftGroupService)
+                    .whenComplete((res, ex) -> raftGroupService.shutdown());
         } catch (NodeStoppingException e) {
             return failedFuture(e);
         }
     }
 
+    private RaftGroupService startOneOffRaftGroupService(PeersAndLearners newConfiguration)
```

Comment:
The helper method has only one usage and is very simple inside. I'd rather use `raftMgr.startRaftGroupService(MetastorageGroupId.INSTANCE, newConfiguration);` instead of helper.

Reply:
Fair, I left it because may be mention "one off" may be important and the method name carry this detail. Should we still left it as is or better remove the method and just left the comment (or even no)? I'm not so sure is it important mention.